ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -14,28 +14,35 @@ namespace ntbs_service.Pages_Search
 {
     public class IndexModel : PageModel
     {
+        private readonly NtbsContext context;
         public ValidationService validationService;
-        public INotificationService service;
+        public INotificationService notificationService;
+        public ISearchService searchService;
         public string CurrentFilter { get; set; }
         public PaginatedList<NotificationBannerModel> SearchResults;
         public string NextPageUrl;
         public string NextPageText;
         public string PreviousPageUrl;
         public string PreviousPageText;
+        public List<Sex> Sexes { get; set; }
         public PaginationParameters PaginationParameters;
 
         [BindProperty(SupportsGet = true)]
         public SearchParameters SearchParameters { get; set; }
+        
         public bool? SearchParamsExist { get; set; }
 
-        public IndexModel(INotificationService service)
+        public IndexModel(INotificationService notificationService, ISearchService searchService, NtbsContext context)

```

Comment:
This method is getting quite involved - can we get a unit test for this?
See `IndexPageTest` for inspiration :)
(Appreciate this might change somewhat when merging with Farman's builder changes, merge probably worth doing first)